### DISCUSSION TITLE
Project 3: Multi-channel Imaging Support

### DISCUSSION THREAD

**Discussion by tarunjain0751** [camicroscope/GSOC#44] [General]
Hi @birm 

I hope you are doing well. I thank you for accepting my proposal for GSoC23. I'm looking forward to seeing what we can accomplish over the course of the summer. Let's schedule a call to discuss the project in more detail and get started on planning out the next steps.

Thanks & regards, 
Tarun Jain

**Comment by tarunjain0751** [comment]
My main GitHub account: @lucifertrj

**Comment by birm** [comment]
Thank you for reaching out! Hopefully you just got an email with a scheduling poll for a welcome call.

For you personally, what is your preferred way to communicate?

**Comment by tarunjain0751** [comment]
Google Meet or Zoom.
